6_habitat.yml

```diff
@@ -0,0 +1,23 @@
+target_default: 6_habitat
+
+packages:
+  - yaml
+
+sources:
+  - src/file_utils.R
+
+targets:
+  6_habitat:
+    depends:
+      - out_data/6_glm2pb0_annual_metrics.csv
+      - out_data/6_glm2pball_annual_metrics.csv
+
+  out_data/6_glm2pb0_annual_metrics.csv:
+    command: update_hash_path(target_name,
+      '../lake-temperature-out/3_summarize/out/annual_metrics_glm2pb0.csv',
```

Comment:
does calling update_hash_path work here? I thought that function was designed for yaml files??

Reply:
Yes! Works for other files, too. Using it over in mntoha also: https://github.com/USGS-R/mntoha-data-release/blob/master/7_habitat.yml#L26-L29

Reply:
you know though ... even though it works maybe we shouldn't use it and should write something else instead. I am going to update

Reply:
Ok, a simple file.copy is all that is needed here. I changed and rebuilt but the resulting CSVs are the same, so the log file didn't change.